config_spec.yaml

```diff
@@ -252,6 +258,13 @@ advanced:
   nullable: false
   nullByDefault: false
   spec:
+    db-bolt-auto-compact:
+      type: boolean
+      name: Compact Database on Startup
+      description: |
+        Performs database compaction on startup. This is necessary to keep disk usage down over time at the cost of
+        having longer startup times.
+      default: true
```

Comment:
This keeps things small to extend the time at which we will run into problems on 32bit. This also keeps the disk footprint down.